README.md

```diff
@@ -45,10 +45,10 @@ Usage is essentially three different steps.
 The test can be started with the following:
 
 ```
-python3 main.py -c ~/path_to_config.yaml -o ~/path_to_report.txt
+python3 main.py -c ~/path_to_config.yaml -r ~/path_to_report.txt
 ```
 
-The `-c` parameter references the configuration file. The `-o` is the location to store the result.
+The `-c` parameter references the configuration file. The `-r` is the location to store the result.
```

Comment:
I think we need to add some documentation on the new parameter you added to the tolerances.  So that team's can understand which tests can have an overridden runtime.

Otherwise thumbs up!

Reply:
makes sense, updated the docs as such. Thanks.